torch/onnx/symbolic_opset9.py

```diff
@@ -362,6 +362,18 @@ def atan(g, self):
     return g.op("Atan", self)
 
 
+def atan2(g, input, other):
+    div_res = div(g, input, other)
+    atan = g.op("Atan", div_res)
+    negpi = torch.tensor(-math.pi, dtype=torch.float32)
+    zero = torch.tensor(0.0, dtype=torch.float32)
+    lessZero = g.op("Less", other, zero)
```

Comment:
camel_case and inline locals that are simple and only used once:

```suggestion
    less_zero = g.op("Less", other, torch.tensor(0.0, dtype=torch.float32))
```